### ISSUE TITLE
FFXVIFix.asi detected as Trojan:Win32/Wacatac.B!ml by Windows Defender

### ISSUE THREAD

**Issue by Cardiacman13** [Lyall/FFXVIFix#52]
Windows Defender detects FFXVIFix.asi as Trojan:Win32/Wacatac.B!ml : 
 
 
![image](https://github.com/user-attachments/assets/65a0d5f8-7443-4864-8314-48f2bf489916)

**Comment by Arron17** [comment]
Yes False Positive with Windows Defender. I just had it, updated the Anti-Virus definitions and now it's stopped detecting it again.

**Comment by Lyall** [comment]
It's a false positive from Windows Defender. Here is a [VirusTotal report](https://www.virustotal.com/gui/file/08c59ec1bf3a215b0fcea303883017e8e771ded20835a3adebcc3bd6bf9a062f/detection/f-08c59ec1bf3a215b0fcea303883017e8e771ded20835a3adebcc3bd6bf9a062f-1726797668) for the latest version (v0.7.9). You can also see the full source code here on GitHub. It's good to practice caution when download stuff online and common sense is the best anti-virus. This fix is safe to use, but if you have any doubts, you can always compile the source code on your own machine locally.

I'll pin this issue so there aren't duplicates made as I've already gone over the false positive report before. Once flagging stops or is removed, I will close this issue.

**Comment by Lyall** [comment]
For some fun I compiled a blank DLL template in Visual Studio and uploaded it to [VirusTotal](https://www.virustotal.com/gui/file/98ac04296e3156f3fc84c6ffe1ff8d0d483b629238378ae276cc650d4fd1c3f7?nocache=1). It gets 3 false positive results despite containing no real code at all.

![image](https://github.com/user-attachments/assets/82d83a24-2694-4b75-8425-3c5fbfe79ea9)
